### ISSUE TITLE
[meta] public logging of #znc

### ISSUE THREAD

**Issue by Mikaela** [znc/znc#1170]
I personally have no opinion, but at least one person has expressed interest and I think it could be discussed to decide what to do.

https://botbot.me/ seems to be popular service for it with easy to remember URL.

**Comment by unknownbike** [comment]
Yes i too support this idea.

**Comment by Mikaela** [comment]
I forgot to say that I have one requirement if public logging happens: **mention in ENTRYMSG "This channel is publicly logged at https://botbot.me/freenode/znc**. Also in topic, but it requires making space and saying that in shorter form such as `Logs: https://botbot.me/freenode/znc`.

**Comment by unknownbike** [comment]
that's cool :)

**Comment by Equinox** [comment]
```it would be a great idea to log this channel, but the thing would be those who would be joining #znc would be running it like 24x7x365 thus no point in logging this channel :D```

Assuming the one person you're talking about showing interest is also the person who made the above comment, they've basically pointed out the main flaw: it is redundant.

Nothing that important happens in #znc (well, the odd bit of drama can be rather entertaining... ) that would require anyone to need to see logs of the channel spanning 24/7/365.

**Comment by Mikaela** [comment]
Logging services generally tend to hide join/part/quit messages and being indexed by search engines it's possible that people searching issues they have could find a solution from public logs. It could also help in case someone doesn't keep logs and needs to find logs from few days ago which was the case of theh person for whom I opened this if I recall correctly.

I think that by thinking about this more I am in support of this and it might not hurt #znc-dev either as I think people with coding problems also use search engines.

**Comment by thenoncoder** [comment]
I completely disagree with this, plus it's useless, and what if someone's personal info gets there, etc, etc.
